src/main/java/com/PhamKornbluhGroup/APIResultData.java

```diff
@@ -0,0 +1,67 @@
+package com.PhamKornbluhGroup;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.nio.charset.StandardCharsets;
+
+public final class APIResultData {
+    private int responseCode;
+    private String responseMessage;
+    private String encoding;
+    private String content;
+
+    public APIResultData() {
+    }
+
+    public void initResultData(HttpURLConnection connection) {
+        StringBuilder builder = new StringBuilder();
+        //b responseStream autocloses
```

Comment:
What are these comments for?

Remember that comments should ideally not explain what the code is doing (that's the code's job), but why the code behaves a certain way.

Reply:
Removed closure explanation. Thanks!